### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by Kelvin-Ben**
# To Do list: list structure
> In this project, I build a simple HTML list of To-Do tasks.
> The list will be styled according to the specifications.
> This simple web page will be built using webpack and served by a webpack dev server.

### CODE REVIEW

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Kelvin-Ben, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters errors. Please fix those errors and request a new review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/makeTasks.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    index: 3,
+  },
+];
+const taskcontainer = document.querySelector('to-do-list');
+tasks = tasks.sort((a, b) => (b.index - a.index));
+export default () => {
+  tasks.forEach((tasks) => {
+    createElement = ('li', 'task-item', 'task-item', `${tasks.description}`, taskcontainer);
+  });
+};
```
Comment (zieeco) [inline src/modules/makeTasks.js]: - Hey, I can't see the populated task as your to-do doesn't show the tasks. please kindly fix this issue.

Screenshot from 2022-09-29 01-18-16

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Kelvin-Ben ,

Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
_Check the comments below,_

- [ ] Please, kindly add a license to make your readme more professional.

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi, @Kelvin-Ben👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️
- The list is now rendered as suggested by the previous reviewer.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



##

###  **Optional Suggestions:**
- N/A

##

Good luck! 🎉 

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**